samples/Playground/Playground.iOS/ViewControllers/Components/GesturesPageViewController.cs

```diff
@@ -21,14 +21,14 @@ public override void ViewDidLoad()
         {
             base.ViewDidLoad();
 
-            var tapCommand = new RelayCommand(() => HighlightView(TapViewContainer));
+            var tapCommand = new RelayCommand<UITapGestureRecognizer>(args => HighlightView(TapViewContainer));
```

Comment:
Usually we name lambda parameters as "x" or as what this parameter is, e.g. "gestureRecognizer".
In this particular case we want to ignore the parameter, so let's use _

Reply:
fixed